content/webapp/test/pages/whats-on.test.tsx

```diff
@@ -4,6 +4,9 @@ import { mountWithTheme } from '@weco/common/test/fixtures/enzyme-helpers';
 // We pull in the page after we've set the config
 import WhatsOnPage from '../../pages/whats-on';
 
+jest.mock('uuid', () => ({
```

Comment:
As we've found out this would most likely be needed on any other `pages` test, I'd add a comment to explain the why and warn future devs that this'll be required in future tests. (Or leave a TODO that's like "Remove when fixed by other package updates" if we do think it's the fix!)